stories/typography.stories.js

```diff
@@ -1,7 +1,10 @@
-import { storiesOf } from "@storybook/vue";
+import { storiesOf, addDecorator } from "@storybook/vue";
 import notes from "@docs/storybook/typography.md";
 
 storiesOf("Typography", module)
+	.addDecorator(() => ({
+		template: '<div style="padding: 15px"><story/></div>',
```

Comment:
Wouldn't it be nice to add this decorator to all stories?
```suggestion
		template: '<div style="padding: 1rem"><story/></div>',
```